components/layout_2020/fragment_tree/base_fragment.rs

```diff
@@ -19,6 +20,7 @@ pub(crate) struct BaseFragment {
 
     /// Flags which various information about this fragment used during
     /// layout.
+    #[ignore_malloc_size_of = "private bitflags type"]
```

Comment:
I think `MallocSizeOf` can be added to bitflags, so that would definitely be preferred here.

Reply:
I added the derive to the declaration but there's an inner private type that I can't modify.